CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+Issues
+------
+
+If you've found a bug or have a great idea for a new feature that will help all users of impress.js, let us know by [adding your suggestion]
+(http://github.com/bartaz/impress.js/issues/new) to the [issues list](https://github.com/bartaz/impress.js/issues).
+
+Guidelines:
+
+* If reporting a bug, please add a [simplified example](http://sscce.org/).
+
+Pull Requests
+-------------
+
+If you have fixed a bug or implemented a feature that you'd like to share, send your [Pull Request](https://help.github.com/articles/using-pull-requests/) against the [master branch]
+(http://github.com/bartaz/impress.js/tree/master). But remember that the team will only accept code that fits the purpose of impress.js
+
+Guidelines:
+
+* Create a new [topic branch](https://github.com/dchelimsky/rspec/wiki/Topic-Branches) for every separate change you make.
+* Make sure impress.js runs successfully in the browser.
```

Comment:
Instead of "the browser", maybe "as much browsers as you can test" would be more restrictive but help more understand the consequences of lack of compatibility